tests/DataFileTest.r

```diff
@@ -46,6 +46,10 @@ test.deleteNonExistantFile <-function() {
 
 test.fileCreationAndDeletion <- function() {
   client <- getAlgorithmiaClient(Sys.getenv("ALGORITHMIA_API_KEY", unset=NA))
+  dataDir <- client$dir("data://.my/largeFiles")
```

Comment:
Making these tests more robust by ensuring that they have the directory they need beforehand.